build/nl_server/download_nl_files.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Function to download all the nl files in catalog.yaml from gcs"""
+
+from nl_server import config_reader
+from nl_server.config import ModelType
+from nl_server.config import StoreType
+from nl_server.embeddings import DOCKER_DATA_FOLDER_PATH
```

Comment:
can we use the new function get_download_root() to get the path here?

Reply:
I think it makes more sense to use the DOCKER_DATA_FOLDER_PATH here because the idea is that: get_download_root() will determine whether to use DOCKER_DATA_FOLDER_PATH or /tmp based on the existence of DOCKER_DATA_FOLDER_PATH .. and at this point in the docker build, that folder won't exist yet

Reply:
oh, i see, this is to get that path ready..